### ISSUE TITLE
List of components has not correct kebab-case

### ISSUE THREAD

**Issue by mathe42** [nuxt/components#165]
**Describe the bug**
When adding a component with file `LMap.js` the infered name is (correct) LMap. But the kebab-case name in the .nuxt/components/readme.md is not correct in this example I get
```
- `<LMap>` | `<lmap>` (correct/path/to/file)
```

I would expect `l-map` to be the kebab case.

This is only a mistake in the readme file. The component can be used with `l-map`.

**To Reproduce**
Add a Component with filename e.g. LMap.js.

**Expected behavior**
Correct kebab-case in the .nuxt/components/readme.md file

**Additional context**
Only the readme file has this problem. At runtime evrything works.

**Comment by pi0** [comment]
Hi @mathe42. Actually it is an intended behavior to support Uper+ naming convention and add dash on rising-edges (If converting, it leads `TEST.js` to be `T-E-S-T`. see [scule](https://github.com/unjs/scule)).

You can use `l-map.vue` name to add dash between L and M

**Comment by mathe42** [comment]
OK.

Just tested it and useing <lmap></lmap> dosn't work!

When you look at this (not nuxt) example:

```html
<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.js"></script>

<div id="app1"></div>
<div id="app2"></div>

<script>
  Vue.component('LMap', {
    template: '<p>Hello World!</p>'
  })

  new Vue({
    el: '#app1',
    template: '<lmap></lmap>'
  })

  new Vue({
    el: '#app2',
    template: '<l-map></l-map>'
  })
</script>
```


The first Vue instance will create a Error:

```
[Vue warn]: Unknown custom element: <lmap> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

(found in <Root>)
```

So for Vue the kebab case version of LMap is l-map...

So this seems to be a bigger Problem than I thought.

**Comment by pi0** [comment]
You are right @mathe42. It seems vue uses special hyphen conversation which was wrongly shown in markdown and loader. Fixed in [v2.1.3](https://github.com/nuxt/components/releases/tag/v2.1.3). You can get update by recreating lock file.
